src/platform/forms-system/test/js/utilities/save-in-progress-messages.unit.spec.js

```diff
@@ -0,0 +1,41 @@
+import { expect } from 'chai';
+
+import { inProgressMessage } from '../../../src/js/utilities/save-in-progress-messages';
+
+describe('inProgressMessage()', () => {
```

Comment:
Oh dang. I could have sworn I wrote tests for this already. :confused: Must have been in a previous iteration that I deleted or never committed. Well, thanks for writing these! :raised_hands: